src/main/entity/Promotion.ts

```diff
@@ -168,6 +170,13 @@ export class Promotion {
   @Column()
   restaurantName: string;
 
+  /*
+   * Represents restaurant location from Google Places API
+   * Many promotions can come from the same restaurant and thus have the same restaurantLocation
+   * */
+  @Column()
+  restaurantLocation: string;
+
```

Comment:
How about `restaurantAddress`?

Also now that I think about it, notice how we have a lot of `restaurant___` properties now inside the Promotion entity. Maybe we should make this a one to one association and make a `Restaurant` entity.

Reply:
Yea, I think we can consider creating a new Restaurant entity in the future, but two properties doesn't seem so bad right now.

Reply:
Alrighty sounds good, is the name restaurantAddress okay though rather than restaurantLocation?

Reply:
Yup I can change it!